auraed/src/runtime/cell_service/cells/mod.rs

```diff
@@ -43,6 +46,10 @@ mod cells;
 mod cgroups;
 mod error;
 
+lazy_static! {
+    pub static ref CELLS: Arc<Mutex<Cells>> = Default::default();
```

Comment:
it feels odd to need this to be pub and static.  essentially, it's now a global (fine i guess but i would have thought some ownership of this thing would be good) but _also_ public.  can we at least limit it to the crate?

Reply:
On my phone so going off memory. 

I don't think I exposed it all the way out, but I'll make the change either way, so it doesn't get accidentally introduced.